packages/contentstack-bulk-publish/src/producer/cross-publish.js

```diff
@@ -210,7 +210,7 @@ async function getSyncEntries(
       if (variantsFlag) {
         for (let index = 0; index < entriesResponse?.items?.length; index++) {
           let variants = [];
-          variants = await getVariantEntries(stack, entries[index].content_type_uid, entries, index, queryParamsObj);
+          variants = await getVariantEntries(stack, entriesResponse.items[index].content_type_uid, entriesResponse, index, queryParamsObj);
```

Comment:
@sunil-lakshman , [Minor] instead of writing redundant entriesResponse.items[index]. I think its better to store like this
const entries = entriesResponse.items[index];
  variants = await getVariantEntries(stack, entries.content_type_uid, entriesResponse, index, queryParamsObj);